packages/accordions/src/elements/accordion/components/Panel.tsx

```diff
@@ -55,16 +53,16 @@ const PanelComponent = forwardRef<HTMLElement, HTMLAttributes<HTMLElement>>((pro
 
   return (
     <StyledPanel
-      {...getPanelProps<PanelProps>({
-        role: null,
+      {...(getPanelProps({
+        role: null as any,
         ref: mergeRefs([panelRef, ref]),
-        index,
+        value: sectionValue,
         isBare,
         isCompact,
         isExpanded,
         isAnimated,
         ...props
-      })}
+      } as any) as any)}
```

Comment:
I think `{...getPanelProps() as HTMLAttributes<HTMLElement>}` would be preferable over the use of `any` here.